### PR TITLE
Update dial-voice-reference.md

### PR DESCRIPTION
**Pull request by CLYVR**
Consolidate supported languages to single location.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/dial-voice-reference.md [hunk 1]
```diff
@@ -95,33 +95,9 @@ Callers can say names in the following formats:
...
-|Chinese (ZH)  |French (FR)  |
-|English (AU)  |German (DE)  |
-|English (CA)  |Italian (IT)  |
-|English (IN)  |Japanese (JP)  |
-|English (UK)  |Portuguese (BR)  |
-|English (US)  |Spanish (ES)  |
-|French (CA)   |Spanish (MX)  |
-
-The following voice commands are available in the 14 languages supported for speech recognition:
+Language support for text-to-speech and speech recognition is available in these [supported languages] (create-a-phone-system-auto-attendant-languages.md).
```
Comment (JohanFreelancer9) [inline Teams/dial-voice-reference.md:98]:
```suggestion
Language support for text-to-speech and speech recognition is available in these [supported languages](create-a-phone-system-auto-attendant-languages.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@CLYVR - please remove the space for the link to work.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**